File: floor.cpp
#include<iostream>

using namespace std;

int main(){
    int t;
    cin>>t;
    while(t--){
        long long n, k;
        cin>>n>>k;
        long long res = -1;
        for(int i=0;i<n;i++){
            long long x;
            cin>>x;
            if(x<=k)
                res=i+1;
        }
        cout<<res<<endl;
    }
    return 0;
}

File: floor.py
t = int(input())

for v in range(t):
    n, k = input().split()
    n, k = int(n), int(k)
    res = -1
    a = list(map(int, input().split()))
    for i in range(n):
        if(a[i]>k):
            break
        res=i+1
    print(res)